cmd/fossa/flags/flags.go

```diff
@@ -49,7 +49,7 @@ var (
 	Endpoint        = "endpoint"
 	EndpointF       = cli.StringFlag{Name: Short(Endpoint), Usage: "the FOSSA server endpoint (default: 'https://app.fossa.com')"}
 	Title           = "title"
-	TitleF          = cli.StringFlag{Name: Short(Title), Usage: "the title of the FOSSA project. (default: the project name)"}
+	TitleF          = cli.StringFlag{Name: Short(Title), Usage: "the title of the FOSSA project (applies ONLY to new projects) (default: the project name)"}
```

Comment:
[nit] I would not emphasize "only" here - I think the emphasis makes this look more alarming than it actually is.